salt/grains/ssh_sample.py

```diff
@@ -12,6 +12,8 @@
 
 
 def __virtual__():
+    if salt.utils.platform.is_windows():
+        return False, "ssh_sample: Not available on Windows"
     try:
         if (
```

Comment:
Why is this check not sufficient enough to have this skipped on windows?

Reply:
@dwoz, 
I would have guessed that this check is sufficient enough,
but testing proofed me wrong.

Initially I wanted to suggest to rely on `is_proxy()` instead of adding `is_windows()`.

Reply:
I think it would be worth while figuring out why the check isn't working instead of working around the fact that it isn't working.

Reply:
@dwoz, I'm glad we agree.